pom.xml

```diff
@@ -36,17 +36,17 @@
 		<uiframeworkVersion>3.19.0</uiframeworkVersion>
 		<registrationcoreVersion>1.10.0</registrationcoreVersion>
 		<registrationappVersion>1.20.0</registrationappVersion>
-		<eventVersion>2.9.0-SNAPSHOT</eventVersion>
-		<idgenVersion>4.6.0</idgenVersion>
+		<eventVersion>2.8.0</eventVersion>
+		<idgenVersion>4.6.0-SNAPSHOT</idgenVersion>
```

Comment:
Why did you change the above to snapshot?

Reply:
For event module, latest changes are in the current version, No significant changes have been made since last release. 
For Idgen module: it has not been changed  in this PR , Am still looking into it

Reply:
You do not seem to have understood my question. Can you look at it again together with the changes in this pull request?

Reply:
haha @sherrif10 can you look again more closer ??

Reply:
really  >><idgenVersion>4.6.0-SNAPSHOT</idgenVersion>   ??

Reply:
oops sorry , i resolved that